examples/base_and_running_difference_mapcubes.py

```diff
@@ -0,0 +1,61 @@
+"""
+==================================================
+Base Difference and Running Difference in Mapcubes
+==================================================
+
+This example illustrates how to do base and running differencing with a MapCube.
+Base differencing uses a fixed reference point when compared to running difference.
+"""
+
+import matplotlib.pyplot as plt
+import matplotlib.colors as colors
+import matplotlib.cm as cm
+
+import sunpy.map
+import sunpy.physics.differential_rotation as diffrot
+from sunpy.data.sample import AIA_193_CUTOUT01_IMAGE, AIA_193_CUTOUT02_IMAGE, AIA_193_CUTOUT03_IMAGE
+
+###############################################################################
+# We create the MapCube using the AIA_193_CUTOUT sample data.
+# To create a MapCube, we can call Map directly but add in a keyword to output a MapCube instead.
+aiamapcube = sunpy.map.Map(AIA_193_CUTOUT01_IMAGE, AIA_193_CUTOUT02_IMAGE,
+                           AIA_193_CUTOUT03_IMAGE, cube=True)
+
+############################################################################
+# We first loop through all the maps in the aiamapcube and differentially rotate each map
+# in the mapcube to the time of the base map in case of base difference.
+# We then subtract each map data from the base map and store all such difference maps in a list.
+# In case of running difference, we loop through all the maps in the aiamapcube and differentially rotate each map
+# in the mapcube with respect to the previous map.
+# We then subtract each map data from the previous map and store all such difference maps in a list.
```

Comment:
This paragraph needs a bit of a rewrite. I think it's a bit confusing as it stands.

Reply:
update the comments. Let me know if its fine